### PR TITLE
do not allow module name start with number

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: src/common/utility.ts [hunk 1]
```diff
@@ -566,6 +566,10 @@ export class Utility {
         if (name.match(/[^a-zA-Z0-9\_]/)) {
             return "The name must contain only alphanumeric characters or the symbol _";
         }
+        if (name.match(/^\d/)) {
```
Comment (adashen) [inline src/common/utility.ts]: instead of do regex match two times. Can we just match with the legal pattern once?

Comment (SLdragon) [reply src/common/utility.ts]: Yes, however, if we use one pattern, the error message maybe not accurate

Comment (adashen) [reply src/common/utility.ts]: How about error message: "The name cannot start with number and must contain  only alphanumeric characters or the symbol _"

Comment (SLdragon) [reply src/common/utility.ts]: Thanks, according to the schema, the validate regex would be /^[a-zA-Z][a-zA-Z0-9_.-]*$/

**Comment by SLdragon** [comment]
We have already updated the json schema store to support  module name start with number, so close this pull request

**Pull request closed without merging**